juniper_official/Chassis/check-failover-init-error.rule

```diff
@@ -0,0 +1,60 @@
+/*
+ * Check failover Initialization error
+ *
+ * Checks if Ksyncd (Junos Kernel State Synchronization daemon) has reported an
+ * initialization error or not.
+ *
+ * Ksyncd cleans up the FIB states on replicated routing engine before it
+ * starts to resync the FIB states from the Master routing engine. It is possible
+ * for this cleanup to fail in certain scenarios. But, if these cleanup failures
+ * are persistent and are seen for more than 30 minutes, it can comprise high
+ * availability feature and should be reported.
+ *
+ * Rebooting backup RE should help in recovery.
+ */
+iceberg {
+    topic chassis.gres {
+        rule check-failover-init-error {
+            synopsis "Check failover Initialization error";
+            description "Check if Ksyncd (Junos Kernel State Synchronization daemon) has reported an initialization error";
+            sensor gres-error-sensor {
+                open-config {
+                    sensor-name /junos/chassis/gres;
+                    frequency 15m;
+                }
+            }
+            field gres-error-state {
+                sensor gres-error-sensor {
+                    path /junos/chassis/gres/error-state;
+                }
+                type string;
+                description "This field captures failover/GRES errors such as initialization error";
+            }
+            trigger report-gres-init-error {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed